Diagnostic/DistroSpecific.py

```diff
@@ -205,6 +205,11 @@ def install_lad_mdsd(self):
     def remove_lad_mdsd(self):
         return self.log_run_get_output('dpkg -P lad-mdsd')
 
+    def patch_lad_omi(self, openssl_ver):
+        # openssl_ver must be 100 or 110
+        cmd = "dpkg -i --refuse-downgrade omi-*.ssl_OPENSSLVER.ulinux.x64.deb".replace("OPENSSLVER", openssl_ver)
```

Comment:
Why is this wildcarded vs hardcode `1.6.8-1`?

Reply:
Following pattern used in install_lad_mdsd. In case we would need to update/rebuild again, one less place to touch the code (it will just install whatever version is in the extension zip).